interface/super/edit_layout.php

```diff
@@ -565,7 +565,7 @@ class='optin noselect' style='width:100%' />";
 
         echo "<input type='text' name='fld[$fld_line_no][list_id]'  id='fld[$fld_line_no][list_id]' value='" .
         htmlspecialchars($linedata['list_id'], ENT_QUOTES) . "'".$type.
-        " size='6' maxlength='30' class='optin listid' style='width:100%;cursor:pointer'".
+        " size='6' maxlength='100' class='optin listid' style='width:100%;cursor:pointer'".
```

Comment:
After address the maxlength at above lines, then this should be ready for the codebase.